sse2neon.h

```diff
@@ -5078,6 +5081,26 @@ FORCE_INLINE __m128 _mm_blendv_ps(__m128 a, __m128 b, __m128 mask)
                                             vreinterpretq_f32_m128(a)));
 }
 
+// Blend packed double-precision (64-bit) floating-point elements from a and b
+// using mask, and store the results in dst.
+// https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_blendv_pd
+FORCE_INLINE __m128d _mm_blendv_pd(__m128d _a, __m128d _b, __m128d _mask)
+{
+#if defined(__aarch64__)
+    uint64x2_t mask =
+        vreinterpretq_u64_s64(vshrq_n_s64(vreinterpretq_s64_m128d(_mask), 63));
```

Comment:
The code is the same as arm 32-bit.
Extract them.

Reply:
The 64-bit implementation uses `vbslq_f64`
The 32-bit implementation uses `vbslq_u64`
Should I unify them to `vbslq_u64`?

Reply:
Using `vbslq_f64` is fine.

Reply:
I can extract this line only
```
uint64x2_t mask =
        vreinterpretq_u64_s64(vshrq_n_s64(vreinterpretq_s64_m128d(_mask), 63));
```

Reply:
That is what I meant.